slack-api-client/src/main/java/com/slack/api/socket_mode/impl/SocketModeClientJavaWSImpl.java

```diff
@@ -219,17 +219,29 @@ public UnderlyingWebSocketSession(URI serverUri, Map<String, String> httpHeaders
             // FIXME: the proxy settings here may not work
             SlackConfig slackConfig = smc.getSlack().getHttpClient().getConfig();
             Map<String, String> proxyHeaders = slackConfig.getProxyHeaders();
+            if (proxyHeaders == null) {
```

Comment:
As the getProxyHeaders() method can return null value, we have a null check like we do in the default SocketModeClientTyrusImpl